src/group_theory/free_abelian_group.lean

```diff
@@ -32,10 +32,8 @@ Here we use the following variables: `(α β : Type*) (A : Type*) [add_comm_grou
 group it is `α →₀ ℤ`, the functions from `α` to `ℤ` such that all but finitely
 many elements get mapped to zero, however this is not how it is implemented.
 
-* `lift (f : α → A) : free_abelian_group α →+ A` : the group homomorphism induced
-by the map `f`.
-
-* `hom_equiv : (free_abelian_group α →+ A) ≃ (α → A)` : the bijection witnessing adjointness.
+* `lift : (α → A) ≃ free_abelian_group α →+ A` : the group homomorphism induced
```

Comment:
I'm not sure whether this might be confusing to newcomers.

Reply:
I suppose I could leave the docstring as it was, and just remove the entry about `hom_equiv`?

Reply:
What about: `the adjunction isomorphism between the free abelian group functor and the forgetful functor` or `the bijection witnessing adjointness between the free abelian group functor and the forgetful functor` combined with something like: `for a function `f: \alpha\to A` the unique group homomorphism `free_abelian_group \alpha\to+ A` can be accessed using `lift f` ? This is probably a bit much for experts, but for beginners it is maybe appropriate? Here it is maybe not as relevant, but maybe if one wants to be uniform over all such constructions, one could also explain how to access the restriction of g for a group homomorphism g.

Reply:
I feel like the category theory interpretation is probably not something we want to include in the description - although it would be quite sensible to add cross-links to the actual definitions of the adjunctions (in a follow-up PR, from someone else who actually knows the lexicon to precisely describe these things!)

Reply:
I think, there is a link to the category theory interpretation in the first paragraph of the module doc. Maybe that is sufficient?